tests/components/sun/test_init.py

```diff
@@ -173,10 +173,10 @@ async def test_norway_in_june(hass: HomeAssistant) -> None:
 
     assert dt_util.parse_datetime(
         state.attributes[entity.STATE_ATTR_NEXT_RISING]
-    ) == datetime(2016, 7, 24, 22, 59, 45, 689645, tzinfo=dt_util.UTC)
+    ) == datetime(2016, 7, 25, 23, 26, 30, 480352, tzinfo=dt_util.UTC)
```

Comment:
While this is a significant difference, I think this may be fixing an existing bug.

It is daylight at the time this test is run, so the next event should be a sunset, but this test currently has the next event being a sunrise nearly a full day before the next sunset, which is not possible.

This date is much closer to the timestamp that existed for Astral V1 (which was 2016, 7, 25, 23, 23, 39), before it https://github.com/home-assistant/core/pull/48573 to this during the V2 upgrade.

A [nearby city](https://www.timeanddate.com/sun/norway/tromso?month=7&year=2016) also has the first sunrise of the summer @ July 25, 23:18, which is much closer to this newer value.